web_app/models/immersed_corrosion_models.py

```diff
@@ -43,3 +43,130 @@ def eval_material_loss(self, time):
         material_loss = (0.00006*self.p['C'] + 0.0008)*time + self.p['b']
 
         return material_loss
+
+
+class thermo_nutrient_variability_steel_corrosion_model(immersed_corrosion_model):
```

Comment:
All of these classes should be in the model py files

Reply:
done